docs/source/visualizations.rst

```diff
@@ -0,0 +1,102 @@
+.. _visualizations:
```

Comment:
For the Dataframe Plot and Metric Correlation Plot I only see the inserted pngs, not the output of the .show() can you check on your end if you see it?

Reply:
yeah thats intentional cause we cant actually run the dash apps on the docs server. they should all just say `Dash is running on http://127.0.0.1:8050/` and show the png